.github/workflows/lint.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Install pipenv
         run: pip install pipenv==2023.12.1
       - name: Set up pipenv
-        run: pipenv sync --dev
+        run: pipenv verify && pipenv sync --dev
```

Comment:
I'm unsure if the `&&` here will result in `pipenv sync` silently not being run if the verify fails?

Reply:
I'm not 100% sure, but I assumed if `pipenv verify` fails here the whole workflow will fail, similar to the Makefile.